docs/FAQ.md

```diff
@@ -80,3 +78,6 @@ the attributes in the viewed result type as defined by the view and converts
 the viewed result type into the service result type using the appropriate
 constructor.
 
+NOTE: If a result type is defined without any views, a "default" view is added
```

Comment:
Should specify that the default view contains the same attributes as the result type (no need to go into the details of collections at this level).